client/account-recovery/reset-password-email-validation/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+import qs from 'qs';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+
+class ResetPasswordEmailValidation extends Component {
+	parseResetData = ( queryString ) => {
+		return qs.parse( queryString );
+	}
+
+	getQueryString = () => {
+		const queryString = get( window, 'location.search', null );
+
+		if ( ! queryString ) {
+			return null;
+		}
+
+		return queryString.slice( 1 );
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<Card>
+				<p>{ translate( 'Validating …' ) }</p>
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Validating' )` *ES Score: 28*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/account-recovery/ar-through-email)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).